CONTRIBUTING.md

```diff
@@ -1,22 +1,49 @@
 ### Contributing to ECMAScript
-There are a number of ways to contribute to ECMAScript. How to actually contribute depends on what you want to accomplish. In many cases, you may be required to execute a CLA with Ecma. See the final section named *Signing the CLA* for more information on this.
+Contributors to ECMAScript and TC39 are expected to follow our [Code of Conduct](https://tc39.github.io/code-of-conduct/).
 
-Ecma TC39 accepts contributions from non-member individuals who have accepted the TC39 copyright and patent policies. Currently all ECMAScript related technical work is done by the TC39 RF TG (Royalty Free Task Group), for which the following IPR Policies apply:
+If you are not an Ecma member, any non-trivial contributions require signing a legal agreement with Ecma. See the section "Required Legal Agreements" below for details.
 
-  * [Ecma International RF Patent Policy](https://ecma-international.org/memento/Policies/Ecma_Royalty-Free_Patent_Policy_Extension_Option.htm)
-  * [Ecma International Software Copyright Policy](https://ecma-international.org/memento/Policies/Ecma_Policy_on_Submission_Inclusion_and_Licensing_of_Software.htm) ([PDF](https://ecma-international.org/memento/Policies/Ecma_Policy_on_Submission_Inclusion_and_Licensing_of_Software.pdf))
+#### Issues and Pull Requests
+For a small changes to ECMAScript, you can contribute by filing an issue or a pull request against the current text of ECMA-262 standard in this repository.
```

Comment:
"a small changes" -> "a small change"
(Mind you, it leaves open what constitutes "small".)

Reply:
Yes, it is ambiguous! I think we'll just have to decide these things on a case-by-case basis. Maybe we want to document these things in some more detail in the process document, but my previous proposal there was rejected.